MAUI/DataForm/Editors.md

```diff
@@ -0,0 +1,1168 @@
+
+{% tabs %}
+{% highlight C# %}
+
+    dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+    
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormItem.FieldName == "agree" && e.DataFormItem is DataFormCheckBoxItem dataFormCheckBox)
+        {
+            dataFormCheckBox.Color = Colors.Pink;
+        }
+    }
+    
+
+{% endhighlight %}
+{% endtabs %}
+		
+## Switch Editor
+
+In switch editor, [Switch](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/switch?view=net-maui-7.0) is loaded, and DataForm [Switch](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/switch?view=net-maui-7.0) editor supports bool data type property.
+
+To add [Switch](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/switch?view=net-maui-7.0) editor in DataForm, register the editor as DataFormEditorType.Switch  for the required property using the [RegisterEditor]`Link` method.
+
+{% tabs %}
+{% highlight C# %}
+
+       dataForm.RegisterEditor("CellularData", DataFormEditorType.Switch);
+       dataForm.RegisterEditor("AirplaneMode", DataFormEditorType.Switch);
+
+        [Display(Name ="Cellular Data")]
+        public bool CellularData { get; set; } = true;
+
+        [Display(Name = "Airplane Mode")]
+        public bool AirplaneMode { get; set; }
+ 
+
+{% endhighlight %}
+{% endtabs %}
+
+`Image`
+
+#### Changing Switch Editor Thumb color
+
+### Using event
+
+By default switch editor Thumb color is Blue, you can change the Thumb color by using `ThumbColor` property in the `DataFormItem`.
+
+{% tabs %}
+{% highlight C# %}
+
+    dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormItem != null && e.DataFormItem.FieldName == "Agree" && e.DataFormItem is DataFormSwitchItem switchItem)
+        {
+             dataFormSwitchItem. ThumbColor = Colors.Pink;
+        }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+#### Changing Switch Editor Background color
+
+### Using event
+
+By default switch editor background colour is white, you can change the Background color by using `OnColor` property in the `DataFormItem`.
+
+{% tabs %}
+{% highlight C# %}
+
+    dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormItem != null && e.DataFormItem.FieldName == "Agree" && e.DataFormItem is DataFormSwitchItem switchItem)
+        {
+            dataFormSwitchItem. OnColor = Colors.Black;
+        }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+
+
+## Combo Box editor
+
+In the Combo Box editor, the [SfComboBox](https://help.syncfusion.com/maui/combobox)  will be loaded.
+
+### Customizing ItemsSource of SfComboBox
```

Comment:
### Changing the ItemsSource of combo box

Reply:
updated